MiniSim/Service/DeviceService.swift

```diff
@@ -34,6 +31,7 @@ protocol DeviceServiceProtocol {
 
 class DeviceService: DeviceServiceProtocol {
     
+    private static let queue = DispatchQueue(label: "com.MiniSim.DeviceService", qos: .utility)
```

Comment:
Isn't quality of service `utility` too low? Previously it was `.userInitiated` as an example: 

For the `focusDevice()` method user expects for this action to happen as soon as possible as it affects the UI.

Reply:
Hm, make sense. I've updated to be `.userInitiated`.